depthai_nodes/ml/parsers/yolo.py

```diff
@@ -66,6 +79,9 @@ def __init__(
         self.iou_threshold = iou_threshold
         self.mask_conf = mask_conf
         self.n_keypoints = n_keypoints
+        self.anchors = anchors if anchors is not None else [None, None, None]
+        self.bbox_only = bbox_only
+        self.yolo_version = yolo_version
```

Comment:
Same question as above: Do we need yolo version or can this be implied by looking if anchors are supplied or not?

Reply:
Answered [here](https://github.com/luxonis/depthai-nodes/pull/96#discussion_r1787344543).